postprocessing/science/rise_time_from_xdmf.py

```diff
@@ -0,0 +1,69 @@
+# Script to determine the rise time (time the absolute slip rate is above a certain threshold) from seissol fault.xdmf output
+# Output of this script is a binary file. If events = 1, the binary file can be exchanged with the rupture time fault output
+# and opened in paraview without creating a new xdmf file.
+
+import numpy as np
+import seissolxdmf
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(description='calculate rise times from fault xdmf file')
+parser.add_argument('filename', help='path+filename.xdmf')
+parser.add_argument('--threshold', nargs=1, metavar=('threshold'), default=([0.01]), help='specify slip rate threshold' ,type=float)
+parser.add_argument('--dt', nargs=1, metavar=('dt'), default=([0.1]), help='insert time step of xdmf file' ,type=float)
+parser.add_argument('--events', nargs=1, metavar=('events'), default=([1]), help='number of events [1,2] in xdmf file' ,type=int)
+args = parser.parse_args()
+
+if args.events[0] != 1 and args.events[0] != 2:
```

Comment:
better use `choices` in the argparse?
see e.g. https://github.com/SeisSol/SeisSol/blob/master/postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py#L15